fb-install-foreman.bats

```diff
@@ -96,6 +96,7 @@ setup() {
 @test "configure repository" {
   if tIsRedHatCompatible; then
     if [ -n "$FOREMAN_CUSTOM_URL" ]; then
+      # use a temporary repo definition as the foreman-release EVR isn't known
       cat > /etc/yum.repos.d/foreman-custom.repo <<EOF
```

Comment:
It's used on line 107 to install foreman-release from the repo under test.

Alternatively we could install the yum tmprepo plugin and use that to `yum install` the release RPM without creating a repo definition on disk, but that's another step and introduces more non-standard software onto the host.

In our yum.theforeman.org repos we have a `foreman-release.rpm` symlink which is used in our quickstart docs, but this is created [during the push process](https://github.com/theforeman/foreman-infra/blob/master/deploy/config/deploy/yum.rb#L49) and isn't present in our repos generated on Koji.